cmd.go

```diff
@@ -192,8 +192,24 @@ func (ctx *Context) Verbosef(format string, params ...interface{}) {
 	}
 }
 
+// Errorf allows for the logging of error messages from a command's
+// context. This should be used for errors which cause a command to fail.
+// Usually these errors are logged by returning them in Run, but that is
+// not always sufficent. For instance, if the client has performed multiple
+// actions
+func (ctx *Context) Errorf(format string, params ...interface{}) {
+	//Here we use the Loggo.logger method `Logf` as opposed to
+	//`logger.Errorf` to avoid introducing an additional call stack
+	//level (since `Errorf` calls `Logf` internally). This is done so
+	//that this function can produce more accurate source location
+	//debug information.
```

Comment:
Can we format this correctly?

Reply:
This was a copy and past job. Fixed here and elsewhere